release-notes-1.9.0.md

```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
 
 #### Known Issues
+
+- None
```

Comment:
I think this needs renamed to release-notes-1.8.2.md

Reply:
done